src/node_worker.cc

```diff
@@ -368,6 +404,12 @@ void Worker::Run() {
       if (exit_code_ == 0 && !stopped)
         exit_code_ = exit_code;
 
+  active_worker_count--;
```

Comment:
There may be situations in which a `Worker` object is created but doesn’t get to run until here. This should go into the `Worker` class destructor.

Reply:
Yeah, I considered that. I moved it here because of the `active_worker_count_warning` switch (moving the decrement here allows us to detect subsequent exceeds earlier), but if we limit the warning to only once per process then moving the decrement to the destructor is fine.